decidim-generators/lib/decidim/generators/app_templates/initializer.rb

```diff
@@ -1,488 +0,0 @@
-
-  # Admin admin password configurations
-  # config.admin_password_strong = Decidim::Env.new("DECIDIM_ADMIN_PASSWORD_STRONG", true).present?
-
-  # config.admin_password_expiration_days = Decidim::Env.new("DECIDIM_ADMIN_PASSWORD_EXPIRATION_DAYS", 90).to_i
-  # config.admin_password_min_length = Decidim::Env.new("DECIDIM_ADMIN_PASSWORD_MIN_LENGTH", 15).to_i
-  # config.admin_password_repetition_times = Decidim::Env.new("DECIDIM_ADMIN_PASSWORD_REPETITION_TIMES", 5).to_i
-
-  # Delete inactive users configuration
-  # config.delete_inactive_users_after_days = Decidim::Env.new("DECIDIM_DELETE_INACTIVE_USERS_AFTER_DAYS", 365).to_i
-  # config.minimum_inactivity_period = Decidim::Env.new("DECIDIM_MINIMUM_INACTIVITY_PERIOD", 30).to_i
-  # config.delete_inactive_users_first_warning_days_before = Decidim::Env.new("DECIDIM_DELETE_INACTIVE_USERS_FIRST_WARNING_DAYS_BEFORE", 30).to_i
-  # config.delete_inactive_users_last_warning_days_before = Decidim::Env.new("DECIDIM_DELETE_INACTIVE_USERS_LAST_WARNING_DAYS_BEFORE", 7).to_i
-
-  # Additional optional configurations (see decidim-core/lib/decidim/core.rb)
-  # config.cache_key_separator = Decidim::Env.new("DECIDIM_CACHE_KEY_SEPARATOR", "/").to_s
-  # config.cache_expiry_time = Decidim::Env.new("DECIDIM_CACHE_EXPIRATION_TIME", "1440").to_i.minutes
-  # config.stats_cache_expiry_time = Decidim::Env.new("DECIDIM_STATS_CACHE_EXPIRATION_TIME", 10).to_i.minutes
-  # config.expire_session_after = Decidim::Env.new("DECIDIM_EXPIRE_SESSION_AFTER", "30").to_i.minutes
-  # unless Decidim::Env.new("DECIDIM_ENABLE_REMEMBER_ME", "auto").default_or_present_if_exists.to_s == "auto"
-  #   config.enable_remember_me = Decidim::Env.new("DECIDIM_ENABLE_REMEMBER_ME", "auto").default_or_present_if_exists
-  # end
-
-  # config.session_timeout_interval = Decidim::Env.new("DECIDIM_SESSION_TIMEOUT_INTERVAL", "10").to_i.seconds
-  # config.follow_http_x_forwarded_host = Decidim::Env.new("DECIDIM_FOLLOW_HTTP_X_FORWARDED_HOST").present?
-  # config.maximum_conversation_message_length = Decidim::Env.new("DECIDIM_MAXIMUM_CONVERSATION_MESSAGE_LENGTH", "1000").to_i
-  # config.password_similarity_length = Decidim::Env.new("DECIDIM_PASSWORD_SIMILARITY_LENGTH", 4).to_i
-  # config.denied_passwords = Decidim::Env.new("DECIDIM_DENIED_PASSWORDS").to_array(separator: ", ")
-  # config.allow_open_redirects = Decidim::Env.new("DECIDIM_ALLOW_OPEN_REDIRECTS").present?
-  # config.enable_etiquette_validator = Decidim::Env.new("DECIDIM_ENABLE_ETIQUETTE_VALIDATOR", true).present?
-end
-
-# if Decidim.module_installed? :api
-#   Decidim::Api.configure do |config|
-#     config.schema_max_per_page = Decidim::Env.new("API_SCHEMA_MAX_PER_PAGE", 50).to_i
-#     config.schema_max_complexity = Decidim::Env.new("API_SCHEMA_MAX_COMPLEXITY", 5000).to_i
-#     config.schema_max_depth = Decidim::Env.new("API_SCHEMA_MAX_DEPTH", 15).to_i
-#     config.disclose_system_version = Decidim::Env.new("DECIDIM_API_DISCLOSE_SYSTEM_VERSION").present?
```

Comment:
This one isn't documented at `docs/modules/configure/pages/environment_variables.adoc`, can you add it please?